docs/source/tutorial.rst

```diff
@@ -246,6 +246,41 @@ Let's try to change the contract using ``token`` received on `Transfer` creation
 .. include:: tutorial/modify-contract.http
    :code:
 
+Contract credentials change
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to get rights for future contract editing, you need to generate Transfer and then apply Transfer using this view ``POST: /contracts/{id}/ownership``, where ``id`` stands for contract id.
```

Comment:
In order to get rights for future contract editing, you need to generate `Transfer` object and then apply this `Transfer` using the endpoint ``POST: /contracts/{id}/ownership``, where ``id`` stands for contract ID.